ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml

```diff
@@ -108,6 +108,32 @@ tests:
     - chain: ipi-install-heterogeneous-day2-kerneltype
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-byo-iam-profile-arm-f7
```

Comment:
I guess we also need a z-stream upgrade for the new profile.

Reply:
is it correct? `aws-ipi-byo-iam-profile-arm-f28` -> `4.17__multi-nightly-4.17-upgrade-from-stable-4.17.yaml`

Reply:
Right.

Reply:
BTW, do you think we need to merge the install profile with `aws-ipi-byo-iam-role` ?

Reply:
The current logic is more straightforward I think, especially for post-check steps, otherwise we may have more conditional branches and it's hard to read.